EE_API_automation/pytest/src/test_planner.py

```diff
@@ -744,8 +745,9 @@ def test_load_list_view_execution(self):
             assert r.status_code == 200
             assert helpers.extract_header("Content-Encoding", r) == "gzip"
 
-    class TestClass_Teardown(object):
-        def test_teardown(self, cleanup):
+    class TestClass_AgileTeardown(object):
```

Comment:
Why do we have a sperate teardown class? Couldn't this be part of the main test suite?

Reply:
I can opt for a single teardown function at suite level, but then once the SDD tests are done, I don't need it anymore so I chose to delete spaces as soon as the need is met.